src/main/java/uk/ac/ebi/eva/commons/models/metadata/AnnotationMetadata.java

```diff
@@ -17,6 +17,7 @@
 package uk.ac.ebi.eva.commons.models.metadata;
 
 import org.springframework.data.mongodb.core.mapping.Document;
+import org.springframework.util.Assert;
 
 @Document
 public class AnnotationMetadata {
```

Comment:
This is a Mut designed class I don't have any clue at all about it. My only contribution was to assert that vepVersion and cacheVersion are required.